content/automate/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Methods.class/__methods__/check_provisioned.rb

```diff
@@ -47,10 +47,11 @@ def prepare_and_call_refresh_provider(service)
 
               def refresh_provider(service)
                 provider = service.orchestration_manager
+                stack    = service.orchestration_stack
 
-                @handle.log("info", "Refreshing provider #{provider.name}")
+                @handle.log("info", "Refreshing provider '#{provider.name}' and stack '#{stack.name}'")
```

Comment:
@bzwei Any chance that `stack` could be `nil` here and `stack.name` or the `stack.ems_ref` below could fail?

Reply:
When the state machine comes to refresh I think the stack must exist.